core/src/test/java/org/pentaho/di/core/util/EnvUtilTest.java

```diff
@@ -30,9 +30,7 @@
 
 import java.util.Locale;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
```

Comment:
We should avoid import *. This is for sure IDE auto stuff...

Reply:
Fixed